### ISSUE TITLE
parallel proof generation and websnark integration

### ISSUE THREAD

**Issue by therealyingtong** [rollupnc/rollupNC_coordinator#2]
- script to submit inputs to SNARK in parallel
- integrate websnark to generate proofs quickly (already done in https://github.com/therealyingtong/rollupnc_ui/blob/master/src/components/Withdraw.vue#L195, for reference)
